ckanext/scheming/helpers.py

```diff
@@ -339,6 +339,8 @@ def scheming_display_json_value(value, indent=2):
         serialized.
     :rtype: string
     """
+    if isinstance(value, basestring):
+        return value
```

Comment:
@amercader I still don't understand this special case. When would the helper be passed an already-serialized JSON string, and if we need this special case for some reason it should at least be part of the docstring.

"Returns the object passed serialized as a JSON string with sorted keys and indent given, unless a unicode or bytes value is passed, those are returned unchanged" . This description just leaves me asking "why?? what a strange function!"

Reply:
oh, it's because of form errors, isn't it.. hmm

Reply:
If that's the reason you can do `if value is string` in the jinja2 form snippet instead and we don't end up with this odd special case in the helper